packages/telegram-stamper/src/index.ts

```diff
@@ -0,0 +1,185 @@
+/// <reference lib="dom" />
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import { TelegramStamperError } from "./errors";
+
+declare global {
+  interface Window {
+    Telegram: any;
```

Comment:
can we be more specific than "any" here? Let's bring this package: https://www.npmjs.com/package/telegram-webapps?

It has all the types we need for CloudStorage:
https://github.com/DavisDmitry/telegram-webapps/blob/d935b6aec22d9361982ba54667eb56faf76b474a/src/index.d.ts#L970

Reply:
So this isnt an "official" Telegram package, and I've been going back and forth about including it. Telegram unfortunately only officially offers `<script src=telegram-web-app.js />`. Its really behind, it terms of how modern apps are developed, and other developers have created much better abstractions like the one you linked. However now were depending on random devs to keep it up to date rather than using the official Telegram source. Given that do you have a strong opinion here?

Reply:
I see, that makes sense. Ill look into that,